packages/klighd-core/src/views-rendering.tsx

```diff
@@ -1058,9 +1062,9 @@ export function getKRendering(datas: KGraphData[], context: SKGraphModelRenderer
             continue
         if (data.type === K_RENDERING_REF) {
             if (context.kRenderingLibrary) {
-                const id = (data as KRenderingRef).renderingId
+                const id = (data as KRenderingRef).properties['klighd.lsp.rendering.id'] as string
                 for (const rendering of context.kRenderingLibrary.renderings) {
-                    if ((rendering as KRendering).renderingId === id) {
+                    if ((rendering as KRendering).properties['klighd.lsp.rendering.id'] as string === id) {
                         context.boundsMap = (data as KRenderingRef).calculatedBoundsMap
```

Comment:
What is a case where this situation is reached? @NiklasRentzCAU

Reply:
The KGraphDiagramSynthesis uses a rendering library for all default renderings of nodes and edges, if activated in the synthesis options. You can try that out with some .kgt file